gate-core/src/main/groovy/com/netflix/spinnaker/gate/config/ServiceConfiguration.groovy

```diff
@@ -45,6 +46,6 @@ class ServiceConfiguration {
   }
 
   Service getService(String name) {
-    services[name]
+    (services + integrations)[name]
```

Comment:
I kept the `integrations` structure the same as `services` in `gate.yml` so that this code works